Documentation/operations/metrics.rst

```diff
@@ -190,93 +190,92 @@ Exported Metrics
-``endpoint_regeneration_time_stats_seconds`` ``scope``                                          Endpoint regeneration time stats
-``endpoint_state``                           ``state``                                          Count of all endpoints
-============================================ ================================================== ========================================================
+============================================ ================================================== ========== ========================================================
+Name                                         Labels                                             Default    Description
+============================================ ================================================== ========== ========================================================
+``endpoint``                                                                                    Enabled    Number of endpoints managed by this agent
+``endpoint_regenerations_total``             ``outcome``                                        Enabled    Count of all endpoint regenerations that have completed
+``endpoint_regeneration_time_stats_seconds`` ``scope``                                          Enabled    Endpoint regeneration time stats
+``endpoint_state``                           ``state``                                          Enabled    Count of all endpoints
+============================================ ================================================== ========== ========================================================
 
 Services
 ~~~~~~~~
 
-========================================== ================================================== ========================================================
-Name                                       Labels                                             Description
-========================================== ================================================== ========================================================
-``services_events_total``                                                                     Number of services events labeled by action type
-========================================== ================================================== ========================================================
+========================================== ================================================== ========== ========================================================
+Name                                       Labels                                             Default    Description
+========================================== ================================================== ========== ========================================================
+``services_events_total``                                                                     Enabled    Number of services events labeled by action type
```

Comment:
I don't see this in my output locally or in the output you posted to the PR. Could you please double-check all of these so that if it says "Enabled" in the default column, it shows up in the output you posted (and vice versa, if it says "Disabled" in the default column it should not show up in your output)?

Also, the description says "labeled by action type", should the "Labels" column have the name of that label?

Reply:
Hmm, as per my other code reviews below it isn't quite as clear cut that these should say "Enabled" when they show up in your output or "Disabled" if they don't. I ended up going through all of the metrics listed here and double-checking them against my local kind dev environment and the codebase. This one does still need updating to disabled though by the looks.

Reply:
Thanks for taking time out for review @joestringer.
I noticed the metrics of type `Vec` such as https://github.com/cilium/cilium/blob/590388d728de5ae17f950cb8bf1cc694049522d3/pkg/metrics/metrics.go#L1027 don't show up in the list however conforming to codebase I have marked them as `Enabled`. This is true for all other review comments as well.

Reply:
:bulb: Ah curious. I still don't quite understand why they don't actually show up by default though :thinking:

Reply:
Not entirely sure myself. Will continue to dig deeper codebase with this being the first commit ;)
It seemed safe to mark them as `Enabled` for now